external-crates/move/crates/move-analyzer/prettier-move/tests/functions/dot_expression.move

```diff
@@ -0,0 +1,23 @@
+// options:
+// printWidth: 70
+// tabWidth: 4
+
+module prettier::dot_expression {
+    fun dot_expression() {
+        board.place(1, 0); // black move
+        board.assert_score(vector[0, 1, 0]); // empty / black / white
+
+        board.first().second().ultra_long_third();
+        board.start_a_chain().then_call_something().then_call_something_else();
+
+        board.assert_score(vector[0, 1, 0]).assert_score().assert_score(vector[0, 1, 0]);
+        board.assert_state(vector[
+            vector[2, 0, 0, 0],
```

Comment:
Please add a test for index notation when a list of indexes is very long with both short and long index values